hubmap/frontend/src/middleware/index.js

```diff
@@ -0,0 +1,7 @@
+/** *
+ * This exports all required components for middleware.
+ */
+
+export { default as rootReducer } from './rootReducer';
+export { default as store } from './store';
+export { default as PubSubApi } from './PubSub';
```

Comment:
We might want to add redux+pubsub at some point. but I removed it for now, since i don't intend to implement it now. I would like to complete working on this PR for now. So I removed PubSub.js now.